packages/payment-mop-up-job/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@defra-fish/payment-mop-up-job",
```

Comment:
Can you update the link and text in the main project repo README.md as this differs from the name there

Reply:
fixed